index.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const numApproved = 2;
+
+module.exports = robot => {
+	robot.on('pull_request.opened', async context => {
+	    console.log("SOMETHING IS HAPPENING");
+	    // Array of all pull requests and getting the first object in the array
+	    var usableContext = JSON.parse(JSON.stringify(context));
+	    var owner = usableContext["payload"]["pull_request"]["head"]["repo"]["owner"]["login"];
+	    var repo = usableContext["payload"]["pull_request"]["head"]["repo"]["name"];
+		
+		var sha = usableContext["payload"]["pull_request"]["head"]["sha"]
+		if (sha == null) {
+			console.log("SHA IS NULL");
+		} else {
+			setStatus = await context.github.repos.createStatus({
```

Comment:
Add the `context` and `description` values here so the status in the PR can help drive action.